go/vt/vttablet/tabletserver/planbuilder/dml.go

```diff
@@ -470,12 +470,6 @@ func analyzeInsertMessage(ins *sqlparser.Insert, plan *Plan, table *schema.Table
 		return nil, fmt.Errorf("%s must be specified for message insert", col.String())
 	}
 
-	col = sqlparser.NewColIdent("message")
-	num = findCol(col, ins.Columns)
-	if num < 0 {
-		return nil, fmt.Errorf("%s must be specified for message insert", col.String())
-	}
-
```

Comment:
Yeah. It would be a weird use case to just have to id column, but I won't be surprised if someone came up a need like that. So, why enforce anything.